check/format-incremental

```diff
@@ -68,7 +68,7 @@ echo "Comparing against revision '${rev}'." >&2
 
 # Get the _test version of changed python files.
 needed_changes=0
-changed_files=$(git diff --name-only ${rev} -- | grep "\.py$")
+changed_files=$(git diff --name-only ${rev} -- | grep "\.py$" | grep -v "pb2\.py$")
```

Comment:
You should also include the underscore. I can imagine a file accidentally ending in `pb2`, but `_pb2` not so much.

Reply:
done